### PR TITLE
Ato:618: Cloudfront configuration for staging

### PR DESCRIPTION
**Pull request by ethanmills**
## What
Configuration details for cloudfront in staging. This has all been deployed already

### CODE REVIEW

File: ci/cloudfront-orchestration/monitoring/staging/parameters.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "ParameterKey": "CloudfrontDistribution",
```
Comment (hannah-leong) [inline ci/cloudfront-orchestration/monitoring/staging/parameters.json:3]:
```suggestion
    "ParameterKey": "CloudFrontDistribution",
```

**Review by hannah-leong — APPROVED** [review]

File: ci/cloudfront-orchestration/monitoring/staging/parameters.json (hunk 1 above)
Comment (hannah-leong) [reply ci/cloudfront-orchestration/monitoring/staging/parameters.json:3]: Nitpick, feel free to resolve when you want to merge

Comment (ethanmills) [reply ci/cloudfront-orchestration/monitoring/staging/parameters.json:3]: This has to match https://github.com/govuk-one-login/devplatform-deploy/tree/main/cloudfront-monitoring-alarm#:~:text=CloudfrontDistribution

Comment (hannah-leong) [reply ci/cloudfront-orchestration/monitoring/staging/parameters.json:3]: I had a feeling it might be something like that :(

**Pull request merged**